Analysis/R/execute_pipeline.R

```diff
@@ -860,5 +861,5 @@ rmarkdown::render(
     config = paste0(opt[["config"]], ".complete"),
     drop_nodata_years = TRUE
   ),
-  output_file = config[["file_names"]][["report"]]
+  output_file = rprojroot::find_root_file(criterion = ".choldir", "Analysis", "output", config[["file_names"]][["report"]])#QZ: added rprojroot to get the filepath
```

Comment:
What's the intent here?

Reply:
here i was thinking that we could save the report to the output folder and the filename in config only needs to specify the filename without the personalized filepath?